### PR TITLE
feat(css): Add data for `lighting-color`, `flood-color`, `flood-opacity`

### LINKED ISSUE #750: Missing CSS properties defined in SVG
This is a tracker issue so we don't forget to add the data for these props.

- [x] https://github.com/mdn/content/pull/35228 `shape-rendering`
- [x] https://github.com/mdn/content/pull/35170 `stop-opacity`, `stop-color`
- [x] https://github.com/mdn/content/pull/35107 `marker-start`, `marker-mid`, `marker-end`
- [x] https://github.com/mdn/content/pull/35167 `text-anchor`
- [x] https://github.com/mdn/content/pull/34931 `x` and `y`
- [x] https://github.com/mdn/content/pull/35259 `stroke`

__Related issues:__

- [x] https://github.com/mdn/content/issues/34763

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

https://drafts.fxtf.org/filter-effects-1/#FloodOpacityProperty
https://drafts.fxtf.org/filter-effects-1/#FloodColorProperty
https://drafts.fxtf.org/filter-effects-1/#LightingColorProperty

### Related issues and pull requests

Fixes https://github.com/mdn/data/issues/750
Relates to https://github.com/csstree/csstree/issues/50

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -4962,6 +4962,38 @@
...
+    "percentages": "no",
+    "groups": [
+      "Filter Effects"
+    ],
+    "initial": "black",
+    "appliesto": "limitedSVGElementsFloodAndDropShadow",
+    "computed": "specifiedValueClipped0To1",
+    "order": "perGrammar",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flood-color"
```
Comment (skyclouds2001) [inline css/properties.json:4995]:
```suggestion
    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flood-opacity"
```

**Review by pepelsbey — CHANGES_REQUESTED** [review]
Looks good in general, but there are a bunch of lint issues that need to be fixed before the merge. I tried to fix a few, but it seems like it's not enough.

File: l10n/css.json [hunk 2]
```diff
@@ -978,34 +981,49 @@
     "en-US": "{{SVGElement(\"ellipse\")}} and {{SVGElement(\"rect\")}} elements in {{SVGElement(\"svg\")}}"
   },
   "limitedSVGElementsGeometry": {
-    "en-US": "{{SVGElement(\"svg\")}}, {{SVGElement(\"rect\")}}, {{SVGElement(\"image\")}}, and {{SVGElement(\"foreignObject\")}} elements in an <code>svg</code>"
+    "en-US": "{{SVGElement(\"svg\")}}, {{SVGElement(\"rect\")}}, {{SVGElement(\"image\")}}, and {{SVGElement(\"foreignObject\")}} elements in {{SVGElement(\"svg\")}}"
   },
   "limitedSVGElementsPath": {
```
Comment (skyclouds2001) [inline l10n/css.json:986]:
```suggestion
```

File: l10n/css.json [hunk 3]
```diff
@@ -978,34 +981,49 @@
...
-    "en-US": "{{SVGElement(\"path\")}} elements in {{SVGElement(\"svg\")}}"
+    "en-US": "{{SVGElement(\"path\")}} element in {{SVGElement(\"svg\")}}"
   },
```
Comment (skyclouds2001) [inline l10n/css.json:988]:
```suggestion
```

File: l10n/css.json [hunk 4]
```diff
@@ -978,34 +981,49 @@
     "en-US": "{{SVGElement(\"ellipse\")}} and {{SVGElement(\"rect\")}} elements in {{SVGElement(\"svg\")}}"
   },
   "limitedSVGElementsGeometry": {
-    "en-US": "{{SVGElement(\"svg\")}}, {{SVGElement(\"rect\")}}, {{SVGElement(\"image\")}}, and {{SVGElement(\"foreignObject\")}} elements in an <code>svg</code>"
+    "en-US": "{{SVGElement(\"svg\")}}, {{SVGElement(\"rect\")}}, {{SVGElement(\"image\")}}, and {{SVGElement(\"foreignObject\")}} elements in {{SVGElement(\"svg\")}}"
   },
   "limitedSVGElementsPath": {
-    "en-US": "{{SVGElement(\"path\")}} elements in {{SVGElement(\"svg\")}}"
+    "en-US": "{{SVGElement(\"path\")}} element in {{SVGElement(\"svg\")}}"
```
Comment (skyclouds2001) [inline l10n/css.json:987]:
```suggestion
```

File: l10n/css.json [hunk 5]
```diff
@@ -978,34 +981,49 @@
...
   },
   "limitedSVGElementsShapeText": {
     "en-US": "SVG shapes and text content elements"
   },
```
Comment (skyclouds2001) [inline l10n/css.json:991]:
```suggestion
```

**Review by pepelsbey — APPROVED** [review]
Looks good! Thank you :)

**Pull request merged**